test/JDBC/input/col_length-vu-verify.sql

```diff
@@ -0,0 +1,110 @@
+USE babel_3489_test_db;
+GO
+
```

Comment:
Add some tests with regular `col_length` function call (not inside view, procedures etc).
Include negative test cases too.